SETUP/site_registration_protection.php.example

```diff
@@ -54,4 +54,3 @@ function minimum_form_fill_time($real_name, $username, $userpass, $userpass2, $e
     }
 }
 
```

Comment:
It's not important, but it did highlight that this file didn't get included in the standardization because of the extension. I've fixed that, thanks for finding it!